### ISSUE TITLE
Latest ZOS cli fails with Yarn.

### ISSUE THREAD

**Issue by hickscorp** [OpenZeppelin/openzeppelin-sdk#1134]
We're trying to update our `zos` and `zos-lib` dependencies as they are listed as outdated.

We installed `@openzeppelin/upgrades` (For the smartcontracts) and `@openzeppelin/cli` (for the `zos` cli binary) as a *devDependency* using yarn.

Before, calling `yarn zos ...` would work. But now:

```
yarn zos
yarn run v1.16.0
$ /home/doodloo/whatev/node_modules/.bin/zos
/home/doodloo/whatev/node_modules/@resolver-engine/imports-fs/build/index.js:24
    UriResolver: core_2.resolvers.UriResolver,
                                  ^

TypeError: Cannot read property 'UriResolver' of undefined
    at Object.<anonymous> (/home/doodloo/whatev/node_modules/@resolver-engine/imports-fs/build/index.js:24:35)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/doodloo/whatev/node_modules/@openzeppelin/cli/lib/models/compiler/solidity/SolidityProjectCompiler.js:24:22)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/doodloo/whatev/node_modules/@openzeppelin/cli/lib/models/compiler/Compiler.js:17:35)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

We subsequently tried installing the cli globally using `yarn global add ...` but then the same error occurs while running the global binary.

Everything seem to work fine when using NPM to install it globally though.

**Comment by spalladino** [comment]
Thanks for the heads-up @hickscorp!! This had been fixed in https://github.com/OpenZeppelin/openzeppelin-sdk/pull/1059 for 2.4, but for some reason it was not picked into master. It will be included in the 2.5 release of `@oz/cli`.

**Comment by hickscorp** [comment]
Thank you for the answer @spalladino!

I think maybe tagging `2.5` with an `rc` might be a tad too early? I'm closing this issue though.
